v5/getting-started/linux.rst

```diff
@@ -2,20 +2,24 @@
 Installing on Linux
 ===================
 
-Installing the toolchain
-------------------------
+Installing the toolchain for Ubuntu-Based Distributions
+-------------------------------------------------------
 
-1. Follow the instructions found `here <https://launchpad.net/~team-gcc-arm-embedded/+archive/ubuntu/ppa>`_ to add and install the latest version of the GNU Arm Embedded toolchain.
+1. If you are using a version >18.04, you can find the toolchain `here <https://www.ubuntuupdates.org/package/core/disco/universe/base/gcc-arm-none-eabi>`

```

Comment:
I haven't been following the launchpad PPA. Is there a specific reason why it won't work on >=18.04? I'd prefer to have two simple options for installing the toolchain: one for debian-based distributions (like Ubuntu and its derivatives) and one for non-debian based distributions.

Reply:
apparently some things like `add-apt-repository` don't work the same way on pure debian as they do in ubuntu. [found some extra steps](http://www.webupd8.org/2014/10/how-to-add-launchpad-ppas-in-debian-via.html) you appear to have to do in order to get that working